fitbenchmarking/fitbenchmarking
# 745 Pull more of the bounds info into the problem class

#### Description of Work

Fixes [#745](https://github.com/fitbenchmarking/fitbenchmarking/issues/745)

#### Testing Instructions

1. Run Fitbenchmarking with a variety of options are problems and check that results are consistent

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
